spot-client/src/spot-tv/calendars/google.js

```diff
@@ -61,6 +62,20 @@ export default {
 
         return gapi.client.request(calendarEventsEndpoint)
             .then(response => response.result.items)
+            .catch(response => {
+                const formattedError = {
+                    errorCode: response.result.error.errors[0].domain,
```

Comment:
are you sure `response.result.error` will always be available ? what when it fails on the network layer and there's no response ?

Reply:
oh it's actually google api and not fetch